### DISCUSSION TITLE
Db check against stashdb possible?

### DISCUSSION THREAD

**Discussion by becks0815** [stashapp/stash#3545] [Q&A] [answered]
Is there any way to compare the content of my local db with stashdb to check which scene(s) of an actor or a studio is locally missing?

**Comment by DogmaDragon** [comment, marked as answer]
There are a few userscripts that adds checks for scenes you have locally:
- <https://github.com/7dJx1qP/stashdb-userscripts> also adds a Wishlist page on StashDB
- <https://docs.stashapp.cc/add-ons/userscripts/#stash-checker> supports multiple sites, not just StashDB

**Comment by becks0815** [comment]
Thx for the info. No idea why I didn't find it earlier. Tampermonkey works
